packages/context-menu/test/items-touch.test.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from '@esm-bundle/chai';
+import './patch-touch-detection.js';
+import './items.test.js';
+import { isTouch } from '@vaadin/component-base/src/browser-utils.js';
+
+describe('touch in use', () => {
+  it('should have touch in use', () => {
+    expect(isTouch).to.be.true;
```

Comment:
Does this test run `items.test.js` but with touch? If so, please add a comment to indicate that.

Reply:
Decided to drop these (hacky) tests. A separate build with test runs on a touch device would be a better approach.